### ISSUE TITLE
Profile for Sony DSC-RX100M7

### ISSUE THREAD

**Issue by emaimx** [lensfun/lensfun#1147]
I made a profile for the Sony DSC-RX100M7, with distortion, chromatism and vignetting.
[lensfun.zip](https://github.com/lensfun/lensfun/files/4504926/lensfun.zip)

**Comment by Iglmoos** [comment]
Sorry, I am no programmer at all. I just own a Sony RX100 M7 and I am looking for a Lens Correction profile for this camera. I only found this one. Now it is a xml-file. How do I convert it into a lcp-file, that I can use in common RAW-converters like Lightroom or Darktabel?

Regards, Hartmut

**Comment by maricn** [comment]
@Iglmoos came here with the same question, got my answer at https://github.com/lensfun/lensfun/issues/859#issuecomment-485673867

@emaimx I've tried using the attached profile and it works good..

can this be updated in the database? the current "rx100m6 & compatibles" doesn't contain vignetting corrections, and this profile generally seems more precise..

**Comment by Shadetail** [comment]
Was trying to get this to work with RawTherapee and it was quite simple and works great. Using windows I just placed the lensfun.xml into the C:\Program Files\RawTherapee\5.8\share\lensfun\
After restarting RawTherappe the lens profile was automatically applied. Geometric distortion correction seems pretty spot on to me. Vignetting as well though I have no way of confirming its quality. Chromatic aberration correction is disabled by default, and enabling it produces even more aberration in the one image that I tried, so it's good that it's disabled by default I guess.
